cmd/snappy/cmd_info.go

```diff
@@ -75,10 +75,7 @@ func snapInfo(pkgname string, includeStore, verbose bool) error {
 	snap := snappy.ActiveSnapByName(pkgname)
 	if snap == nil && includeStore {
 		m := snappy.NewUbuntuStoreSnapRepository()
-		snaps, err := m.Details(snappy.SplitOrigin(pkgname))
-		if err == nil && len(snaps) == 1 {
-			snap = snaps[0]
-		}
+		snap, _ = m.Snap(pkgname)
```

Comment:
We have an actual error result here, and below we're saying we didn't find it. Sounds like we should instead report the error we got so the user has a chance of telling what's going on.